build-python3.7.cmake

```diff
@@ -0,0 +1,42 @@
+# -- initial cmake cache file for building agains python3.7
+
+
+# Build DLite against python3.7 on a system with a newer interpreter
+# ==================================================================
+#
+# 1. Install python3.7
+# --------------------
+# On Fedora do `dnf install python3.7`.
+#
+# 2. Create virtual environment
+# -----------------------------
+# With virtualenvwrapper installed, do
+#
+#     mkvirtualenv dlite37 -p /usr/bin/python3.7
+#     python3.7 -m pip install -U pip
+#     python3.7 -m pip install -r requirements.txt
+#     python3.7 -m pip install -r requirements_dev.txt
+#
+# 3. Build dlite
+# --------------
```

Comment:
Should there be a step to also activate the new virtualenv, or is this all taken care of in the commands below to use the virtualenv interpreter?

Reply:
Good point. But `mkvirtualenv` automatically activates the new environment, so it is already taken care of.